api/openapi_server/configs/registry.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum, auto
+from typing import Union
+
+from openapi_server.configs.huu_config import HUUConfig
+
+class HUUConfigRegistry(Enum):
```

Comment:
I'm aware of approaches for runtime registration, but decided to favor readability here with an explicit "compile-time" registration system. Since adding a new config type should be an uncommon task, and we expect to have < 10 configs I didn't think the indirection of a runtime based system was worth it.

Let me know if you disagree.

Reply:
Got it. I meant this is known as a factory. That's all.